journal/modernized/31-chapter-30.adoc

```diff
@@ -1,7 +1,8 @@
-== Section XXX.A Supplement
+== Chapter XXX A Supplement
```

Comment:
It ends up looking fine either way, but let's restore the period and just add a space, since it's kind of our standard for chapter number + subtitle, like shown here:

![2019-04-01_10-35-16](https://user-images.githubusercontent.com/7050938/55335913-03e33800-546a-11e9-9660-bae19779bc47.png)

Reply:
Sounds good. I actually tried to look that up but didn't read it apparently because I didn't see the period at the end of "chapter II" in the example and assumed it shouldn't be there. Doh!